### ISSUE TITLE
Better error message when iRODS authentication times out

### ISSUE THREAD

**Issue by tjelvar-olsson** [jic-dtool/dtool-irods#4]
Currently:

```
$ dtool ls irods:///jic_archive
dtool_irods - WARNING - Command failed: ['ils', u'irods:///jic_archive']
dtool_irods - WARNING - ERROR: lsUtil: srcPath /nbi/home/olssont/irods:/jic_archive does not exist or user lacks access permission

ERROR: lsUtil: srcPath /nbi/home/olssont/irods:/jic_archive does not exist or user lacks access permission
```

**Comment by tjelvar-olsson** [comment]
The above may have come from a different error in the code. However, the bug is still a valid one.

**Comment by tjelvar-olsson** [comment]
Actual error message:

```
$ dtool ls irods:///jic_overflow/rg-levi-yant/
dtool_irods - WARNING - Command failed: ['ils', u'/jic_overflow/rg-levi-yant/']
dtool_irods - WARNING - [-]             libnative.cpp:336:native_auth_client_response :  status [CAT_INVALID_AUTHENTICATION]  errno [] -- message [Call to rcAuthResponseFailed.]
 
 failed with error -826000 CAT_INVALID_AUTHENTICATION 
 
[-]            libnative.cpp:336:native_auth_client_response :  status [CAT_INVALID_AUTHENTICATION]  errno [] -- message [Call to rcAuthResponseFailed.]
 
 failed with error -826000 CAT_INVALID_AUTHENTICATION 
```

**Comment by tjelvar-olsson** [comment]
Fixed in 27f7b7c
